draft-ietf-quic-transport.md

```diff
@@ -187,9 +187,9 @@ QUIC:
 
 QUIC packet:
 
-: A complete processable payload with a packet type, encryption level and
-  packet number. Multiple QUIC packets can be encapsulated in a single UDP
-  datagram.
+: A complete processable unit of QUIC consisting of a header, payload, and
+  authentication tag.  Multiple QUIC packets can be encapsulated in a single
```

Comment:
Not all packets have an authentication tag.  A single atomic unit is really the only thing we can say.